frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -24,6 +23,7 @@ import { CloneVirtualMachineDialog } from '../dialogs/cloneVirtualMachineDialog'
 import { getRandStr } from '../utils/utils';
 
 const noConfirmDialogActions: VM_ACTION[] = [VM_ACTION.Start, VM_ACTION.Clone];
+const annotationDialogOverlay = $('.co-overlay');
```

Comment:
This should go in a view file somewhere and should be generalized to `modalOverlay`.

Reply:
Fixed.  `kubevirt-plugin/integration-tests/.../virtualMachine.ts` was the only Protractor code using `annotationDialogOverlay` from the deleted file `frontend/integration-tests/views/modal-annotations.view.ts`.   Added `export const modalOverlay = $('.co-overlay')` to common kubevirt-plugin view file.